packages/ui/ViewManager/tests/ViewManager-specs.js

```diff
@@ -203,4 +202,58 @@ describe('ViewManager', () => {
 		expect(actual).toBe(expected);
 		done();
 	});
+
+	test('should update the View reverseTransition prop.', done => {
+		const subject = mount(
+			<ViewManager noAnimation index={0} duration={0}>
+				<div className="view">View 1</div>
+				<div className="view">View 2</div>
+				<div className="view">View 3</div>
+				<div className="view">View 4</div>
+				<div className="view">View 5</div>
+			</ViewManager>
+		);
+
+		subject.setProps({reverseTransition: true});
+		const actual = subject.find('View').props().reverseTransition;
+
+		expect(actual).toBeTruthy();
+		done();
+	});
+
+	test('should update the View reverseTransition prop to true if it is updated with a smaller index prop.', done => {
+		const subject = mount(
+			<ViewManager noAnimation index={2} duration={0}>
+				<div className="view">View 1</div>
+				<div className="view">View 2</div>
+				<div className="view">View 3</div>
+				<div className="view">View 4</div>
+				<div className="view">View 5</div>
+			</ViewManager>
+		);
+
+		subject.setProps({index: 1});
+		const actual = subject.find('View').props().reverseTransition;
```

Comment:
The selector does not refer to the `className`. Should we remove it or should it use `className` as the selector?

Reply:
`<div className="view">View 1</div>` ? 
See other tests for comparison. The result would be the same as there's only one `children` rendered unless we try to capture in mid-transition. I was pointing out that you added `className`s for each view but not it was never used as part of the test.